### DISCUSSION TITLE
Are there any plans on supporting face detection?

### DISCUSSION THREAD

**Discussion by CombeeMike** [google/model-viewer#3166] [Q&A]
Are there any concrete plans on supporting face detection with the `modelviewer`?
I only found [this ticket](https://github.com/google/model-viewer/issues/1748) (and the related [discussion](https://github.com/google/model-viewer/discussions/2076)) dealing with the question so far.

Follow-up-questions:
* Is this strictly bound to face detection being support by WebXR or can the `modelviewer` somehow polyfill this feature "on its own"?
* Is there any knowledge of plans in WebXR regarding support for face detection? I only found [this ticket](https://github.com/immersive-web/webxr/issues/1150) from Dec 2020.

**Comment by elalish** [comment]
We're getting more questions about this so it may be time to prioritize. I think better than waiting for a WebXR face tracking API would be use the WebXR raw camera access API instead (which is close to shipping in Chrome) and do the ML image processing in JS, since there are some decent solutions to that now. We'll need to think through the best way to expose an API to use for this in MV. Suggestions welcome!
